### ISSUE TITLE
Possible errors when running clinker v0.0.11 with GenBank files from MiBIG

### ISSUE THREAD

**Issue by 0x1d4e** [gamcil/clinker#26]
Hi, thank you for the great tool. I ran into an error when running the latest version of clinker (v0.0.11) with a GenBank file obtained from MiBIG included. The error is as followed:

```python
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/bin/clinker", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/clinker/main.py", line 206, in main
    clinker(
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/clinker/main.py", line 81, in clinker
    globaligner = align.align_clusters(*clusters, cutoff=identity, jobs=jobs)
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/clinker/align.py", line 51, in align_clusters
    aligner.add_clusters(*args)
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/clinker/align.py", line 325, in add_clusters
    self._genes[gene.uid] = gene
AttributeError: 'NoneType' object has no attribute 'uid'
```
It did not happen with an older version (prior to v0.0.9) in another machine or when the MiBIG GenBank file was not included. Could you check what could be the reason?
Thanks a lot

**Comment by gamcil** [comment]
Oh that's weird, could you give me the file or MIBiG accession so I can have a look? Seems like it may not be parsing genes correctly.

**Comment by 0x1d4e** [comment]
I was trying with this gene cluster BGC0002016 from MiBIG (https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1). When I tried with another (random) one, BGC0002015, clinker worked. So probably something to do with the GenBank file of BGC0002016?

**Comment by gamcil** [comment]
Ahh, a gene in that file is annotated as a pseudogene (C5F59_12930) and clinker wasn't saving it. I just released clinker v0.0.12 which should fix this, so see if that works.

**Comment by 0x1d4e** [comment]
That solved the problem, thanks a lot for the (very) quick fix. And thanks again for the great tools.
